fastlane/Matchfile

```diff
@@ -1,6 +1,10 @@
 # This Matchfile has the shared properties used for all signing types
 
 # Store certs/profiles encrypted in Google Cloud
-storage_mode("google_cloud")
-google_cloud_bucket_name("a8c-fastlane-match")
-google_cloud_keys_file(".configure-files/google_cloud_keys.json")
+storage_mode('google_cloud')
+google_cloud_bucket_name('a8c-fastlane-match')
+# Interestingly, Using the path with '~/.configure/...' results in Fastlane
+# failing to find the file.
+SECRETS_ROOT = File.join(Dir.home, '.configure/simplenote-ios/secrets')
+google_cloud_keys_file(File.join(SECRETS_ROOT, 'google_cloud_keys.json'))
+api_key_path(File.join(SECRETS_ROOT, 'app_store_connect_fastlane_api_key.json'))
```

Comment:
While I'm not a fan of spreading configurations across multiple `-file` in Fastlane, it was handy to have this here considering we have three different calls to `match` in this project.